Poker.BE/Poker.BE.Domain.Tests/Game/RoundTests.cs

```diff
@@ -22,9 +22,9 @@ public void RoundTest()
         public void PlayMoveTest()
         {
             //Arrange
-            var player1 = new Player();
-            var player2 = new Player();
-            var player3 = new Player();
+            var player1 = new Player() { Nickname = "test player 1" };
+            var player2 = new Player() { Nickname = "test player 2" };
+            var player3 = new Player() { Nickname = "test player 3" };
             player1.AddMoney(500);
```

Comment:
Like i told you before, I think that we have to add a moneyDeposit in the Player constructor